src/text/raw_text_writer.rs

```diff
@@ -29,14 +26,26 @@ impl RawTextWriterBuilder {
     /// ```
     pub fn new() -> RawTextWriterBuilder {
         RawTextWriterBuilder {
-            // Single space between values
-            space_between_values: String::from(" "),
-            // No indentation
-            indentation: String::new(),
-            // Single space between field names and values
-            space_after_field_name: String::from(" "),
-            // The first value in a container appears next to the opening delimiter
-            space_after_container_start: String::new(),
+            whitespace_config: WhitespaceConfig {
+                ..DEFAULT_WS_CONFIG
+            },
+        }
+    }
+
+    /// Constructs a 'lines' text Ion writer that adds UNIX and human-friendly newlines between
+    /// top-level values.
+    ///
+    /// For example:
+    /// ```ignore
+    /// {foo: 1, bar: 2, baz: 3}
+    /// [1, 2, 3]
+    /// true
+    /// "hello"
+    /// ```
+    // This doesn't solve the problem of final newlines. Should find a way to solve that some day.
```

Comment:
Could you open an issue for this? I think it would be the right behavior for the `lines` and `pretty` formats.

If you'd like to keep this comment, I think we should link to the issue you open for it.

Reply:
Done: https://github.com/amzn/ion-rust/issues/437